resources/knative-serving-crds-init/README.md

```diff
@@ -0,0 +1,6 @@
+# Knative Serving CRDs
+
+## Overview
+
+>**NOTE**: This is an experimental module.
+This chart includes a pre install hook to delete the old CRDs before installing new CRDs.
```

Comment:
```suggestion

This chart includes a pre-install hook used to delete the old CRDs before installing new CRDs.
```
Add a line, otherwise this part is included in the note.